packages/zarf-registry/zarf.yaml

```diff
@@ -11,6 +11,10 @@ variables:
     description: "The size of the persistent volume claim for the registry"
     default: "20Gi"
 
+  - name: REGISTRY_PVC_ACCESS_MODE
+    description: "The access mode of the persistent volume claim for the registry"
+    default: "ReadWriteOnce"
```

Comment:
Can we add back in the switch on HPA enablement? RWX is probably what we will still use for HPA across most clusters for now so RWO should only be if HPA is disabled.

Reply:
Actually thinking about this again, I think this configuration option is fine since it will be more about what the storage plugin supports - but can we keep the default RWX?

Reply:
NVM I misread this when I first saw it - this is probably fine to allow for configuration